DICOMwebBrowser/DICOMwebBrowser.py

```diff
@@ -1090,12 +1091,25 @@ def onCancel(self):
 
 class GoogleCloudPlatform(object):
 
+  def findgcloud(self):
+    gcloudPath = shutil.which('gcloud')
+    if gcloudPath:
+      return gcloudPath
+
+    if sys.platform in ("linux", "darwin"):
+      # Try a second way to find gcloud path
```

Comment:
The rationale for this must be described in more detail (the same way as you did not know how environment variables are set on macOS, other developers will not know it either and therefore may decide remove this seemingly redundant code when doing some refactoring in the future). Something like this:

```
On macOS: gcloud installs itself in the default shell by adding code to `.zshrc` (on older macOS it would be `.bashrc`). So the environment is set and Slicer sees it if you start Slicer via a terminal. But GUI applications launched via the OS don't go through the terminal so they don't get that environment.

On Windows: the gcloud path is in `os.environ['PATH']` so `shutil.which` finds it.

On Linux: ...? -- please describe how it works on linux
```

Reply:
i confirmed this is also an issue on Linux when launching via a `.desktop` file. I added comments describing the issue to the code